### ISSUE TITLE
The virtual undercloud based deployment is broken on master branch

### ISSUE THREAD

**Issue by sdatko** [redhat-performance/jetpack#386]
We observe failures in PerfCI when switching from tag v2.0 to current master.
TL;DR:
– wrong user selected when accessing undercloud in TASK [check if instackenv file exists] (jetpack/tag.yml:5)
– problem with calculating number of nodes in TASK [Adjust compute count] (jetpack/overcloud.yml:72)
– deployment failure after 5 hours, last task is TASK [add hosts to host list] (infrared/plugins/tripleo-overcloud/tasks/add_overcloud_host.yml:40)
—— fatal: [undercloud]: FAILED! => {}
—— The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'ctlplane'

Related: https://github.com/redhat-performance/jetpack/pull/385 (fixes 1st and 2nd listed issue)
